package.json

```diff
@@ -30,6 +30,7 @@
     "babel-preset-latest": "^6.22.0",
     "babel-preset-react": "^6.22.0",
     "eslint": "^3.16.1",
+    "eslint-config-keystone": "^3.0.0",
```

Comment:
@tayiorbeii Looks like this isn't being used and isn't needed since this extends the CRA `react-app` eslint config; was it added by accident?